### DISCUSSION TITLE
devcontainer 镜像内未安装 python3

### DISCUSSION THREAD

**Discussion by liuhaoyang** [alibaba/ilogtail#410] [Help] [answered]
我使用 devcontainer 进行 `make vendor` 时，返回了一个错误：
```
python3 ./external/sync_vendor.py
make: python3: Command not found
make: *** [vendor] Error 127
```

查看 python 版本
```
[root@31ccbafada10 ilogtail]# python --version
Python 2.7.5
```

**Comment by messixukejia** [comment]
python依赖体验确实比较差，短期内改成shell吧。 这里有个issue https://github.com/alibaba/ilogtail/issues/362。

后面去掉vendor这个依赖。

**Comment by messixukejia** [comment, marked as answer]
该pr已去掉了对python的依赖。 https://github.com/alibaba/ilogtail/pull/412/files
